utils/i18n.go

```diff
@@ -36,7 +36,10 @@ func InitTranslations() {
 func GetTranslationsBySystemLocale() i18n.TranslateFunc {
 	locale := model.DEFAULT_LOCALE
 	if userLanguage, err := jibber_jabber.DetectLanguage(); err == nil {
-		locale = userLanguage
+		// only set the system locale if is supported, fallback to DEFAULT_LOCALE
```

Comment:
there was a bug,  I think we need to add the line `locale = model.DEFAULT_LOCALE` in the if block ?